sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md

```diff
@@ -25,19 +25,20 @@ From a developer's perspective, Key Vault APIs accept and return secret values a
 
 For highly sensitive data, clients should consider additional layers of protection for data. Encrypting data using a separate protection key prior to storage in Key Vault is one example.
 
-Key Vault also supports a contentType field for secrets. Clients may specify the content type of a secret to assist in interpreting the secret data when it's retrieved. The maximum length of this field is 255 characters. There are no pre-defined values. The suggested usage is as a hint for interpreting the secret data.
+Key Vault also supports a contentType field for secrets. Clients may specify the content type of secret to assist in interpreting the secret data when it's retrieved. The maximum length of this field is 255 characters. There are no pre-defined values. The suggested usage is as a hint for interpreting the secret data.
+
+Besides, this starter provides the features of supporting multiple Key Vaults, case-sensitive mode of Key Vault names, and using placeholder presenting Key Vault names in the property file.
```

Comment:
`Besides, this starter supports multiple Key Vaults`

Reply:
Could you please update this doc by the intellij's hints?
![image](https://user-images.githubusercontent.com/13167207/132148533-b78f2834-ad63-4a77-b0a8-eab5d227b8e0.png)

Reply:
OK